package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "sonarplanet-services",
+  "version": "1.0.0",
+  "description": "Address tracker",
+  "main": "app.js",
+  "repository": "git@github.com:spacelephant/sonarplanet-services.git",
+  "author": "spacelephant",
```

Comment:
`author` should be @ftreguer :-)
See https://docs.npmjs.com/files/package.json#people-fields-author-contributors for the format.